docker/environment-r-omero.yml

```diff
@@ -0,0 +1,24 @@
+channels:
+- bioconda
+- conda-forge
+- defaults
+dependencies:
+- bioconductor-ebimage=4.13.*
+- maven=3.5.*
+- openjdk=8.0.*
```

Comment:
openjdk is in both requirement file.
I don't know if it is possible to combine requirements file
saying that the apps might not have the same required version over time

Reply:
As discussed, it's easier to keep them separate.